front/app/services/navbar.ts

```diff
@@ -49,16 +57,28 @@ export function removedDefaultNavbarItems() {
   });
 }
 
-export function toggleAllInput({ enabled }: { enabled: boolean }) {
-  return streams.add(`${apiEndpoint}/toggle_all_input`, { enabled });
+export async function toggleAllInput({ enabled }: { enabled: boolean }) {
```

Comment:
I'm a bit confused about what should and shouldn't be in the navbar module. If any navbar toggling functionality should only work if the navbar module is on, these toggle functions should be moved.